bgpd/bgp_vty.c

```diff
@@ -2579,6 +2579,35 @@ DEFUN (no_bgp_network_import_check,
 	return CMD_SUCCESS;
 }
 
+
+DEFUN (bgp_enable_ipv6_linklocal_address_only,
+       bgp_enable_ipv6_linklocal_address_only_cmd,
+       "[no] bgp enable-ipv6-ll-only",
+       NO_STR
+       "BGP specific commands\n"
+       "Set Link-Local Address only\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+
+	int idx = 0;
+	bool yes = 1;
+
+	if (argv_find(argv, argc, "no", &idx))
+		yes = 0;
+
+	if (yes &&
+		!bgp_flag_check(bgp, BGP_FLAG_IPV6_LINK_LOCAL_ONLY))
```

Comment:
is this check is needed? can't we set it idempotently?

Reply:
Yes it is needed boz the same commands code takes care of "no" operations too.